docs/dashboards/data-handling.md

```diff
@@ -0,0 +1,51 @@
+Data Handling
+=============
+
+Highcharts Dashboards has a separate set of classes to handle data.  The
+following classes are used by the Board class to provide its components with
+data to show.
+
+
+
+## DataPool
+
+With the `BoardOptions.dataPool` option you can define as many connections to
```

Comment:
I would describe it as:
`... You can define as many connections to your data sources as you like.`